instrumentation/mongo/lib/opentelemetry/instrumentation/mongo/command_serializer.rb

```diff
@@ -14,10 +14,11 @@ class CommandSerializer
 
         attr_reader :command, :command_name, :collection, :payload
 
-        def initialize(command)
+        def initialize(command, obfuscate)
           @command = command
           @command_name, @collection = command.first
-          @collection = MASK_VALUE unless @collection.is_a?(String) || @collection.is_a?(Integer)
+          @obfuscate = obfuscate
+          @collection = MASK_VALUE if @obfuscate && !(@collection.is_a?(String) || @collection.is_a?(Integer))
```

Comment:
If I recall correctly, the intention on this line was more about marking invalid data as "?" rather than obfuscating sensitive data; the collection name should always be a String AFAIK. Would suggest to revert this line or do something different without `@obfuscate` variable.

Reply:
I personally don't see a reason to set a mask value unless we're obfusacting - happy to change the behavior if that's what's agreed upon by everyone.

Reply:
Done